heft-plugins/heft-jest-plugin/src/JestPlugin.ts

```diff
@@ -176,14 +272,15 @@ export class JestPlugin implements IHeftPlugin<IJestPluginOptions> {
     }
 
     const jestArgv: Config.Argv = {
-      watch: testStageProperties.watchMode,
+      // TODO: Watch mode
+      // watch: testStageProperties.watchMode,
```

Comment:
So, when this PR goes in, will the Jest plugin not support watch mode at all?

Reply:
Right now yes. I want to implement watch/start stuff after getting in a first pass on reviews.